tests/unit/TranslateTest.ts

```diff
@@ -24,71 +26,79 @@ translations.default = {
-        expect(Localize.translate(CONST.LOCALES.ES_ES, 'testKey3')).toBe('Test Word 3');
+        expect(Localize.translate(CONST.LOCALES.ES_ES, 'testKey3' as TranslationPaths)).toBe('Test Word 3');
     });
 
     test('Test when key is not found in default', () => {
-        expect(() => Localize.translate(CONST.LOCALES.ES_ES, 'testKey4')).toThrow(Error);
+        expect(() => Localize.translate(CONST.LOCALES.ES_ES, 'testKey4' as TranslationPaths)).toThrow(Error);
     });
 
     test('Test when key is not found in default (Production Mode)', () => {
         const ORIGINAL_IS_IN_PRODUCTION = CONFIG.IS_IN_PRODUCTION;
+        // @ts-expect-error - We are modifying the CONFIG object for testing purposes
         CONFIG.IS_IN_PRODUCTION = true;
-        expect(Localize.translate(CONST.LOCALES.ES_ES, 'testKey4')).toBe('testKey4');
+        expect(Localize.translate(CONST.LOCALES.ES_ES, 'testKey4' as TranslationPaths)).toBe('testKey4');
+        // @ts-expect-error - We are modifying the CONFIG object for testing purposes
         CONFIG.IS_IN_PRODUCTION = ORIGINAL_IS_IN_PRODUCTION;
     });
 
     it('Test when translation value is a function', () => {
         const expectedValue = 'With variable Test Variable';
         const testVariable = 'Test Variable';
-        expect(Localize.translate(CONST.LOCALES.EN, 'testKeyGroup.testFunction', {testVariable})).toBe(expectedValue);
+        // @ts-expect-error - We are modifying the translations object for testing purposes
+        expect(Localize.translate(CONST.LOCALES.EN, 'testKeyGroup.testFunction' as TranslationPaths, {testVariable})).toBe(expectedValue);
     });
 });
 
 describe('Translation Keys', () => {
-    function traverseKeyPath(source, path, keyPaths) {
-        const pathArray = keyPaths || [];
+    function traverseKeyPath(source: TranslationFlatObject, path?: string, keyPaths?: string[]): string[] {
+        const pathArray = keyPaths ?? [];
         const keyPath = path ? `${path}.` : '';
-        _.each(_.keys(source), (key) => {
-            if (_.isObject(source[key]) && !_.isFunction(source[key])) {
+        Object.keys(source).forEach((key) => {
+            if (typeof source[key as keyof TranslationFlatObject] === 'object' && typeof source[key as keyof TranslationFlatObject] !== 'function') {
+                // @ts-expect-error - We are modifying the translations object for testing purposes
                 traverseKeyPath(source[key], keyPath + key, pathArray);
             } else {
                 pathArray.push(keyPath + key);
             }
         });
+
         return pathArray;
     }
-    const excludeLanguages = [CONST.LOCALES.EN, CONST.LOCALES.ES_ES];
-    const languages = _.without(_.keys(originalTranslations.default), ...excludeLanguages);
+    function arrayDifference(array1: string[], array2: string[]): string[] {
+        return [array1, array2].reduce((a, b) => a.filter((c) => !b.includes(c)));
+    }
```

Comment:
Could you move this function to the top of the file?

Reply:
Actually I listened to @VickyStash's advice and moved it to utils!